### ISSUE TITLE
Can't install module ui (or any vpm module)

### ISSUE THREAD

**Issue by jagtesh** [vlang/v#4710]
**V version:** 
V 0.1.26 50a8373.d7e7113

**OS:** 
macOS Catalina 10.15.4 (19E287)

**What did you do?**
./v install ui

**What did you expect to see?**
The ui package installed.
 
**What did you see instead?**
```
cannot compile ‘/Users/jag/Repos/v/cmd/tools/vpm.v: 
==================
^
/private/var/folders/hc/7dvjblp92bg_kkpnksn4cqvr0000gn/T/v/vpm.tmp.c:13428:22: error: implicit declaration of function 'json__decode_Mod' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
                                Option_Mod mod = json__decode_Mod(json__json_parse(s));
                                                 ^
/private/var/folders/hc/7dvjblp92bg_kkpnksn4cqvr0000gn/T/v/vpm.tmp.c:13428:16: error: initializing 'Option_Mod' (aka 'struct Option') with an expression of incompatible type 'int'
                                Option_Mod mod = json__decode_Mod(json__json_parse(s));
                                           ^     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/private/var/folders/hc/7dvjblp92bg_kkpnksn4cqvr0000gn/T/v/vpm.tmp.c:13460:38: warning: integer literal is too large to be represented in a signed integer type, interpreting as unsigned [-Wimplicitly-unsigned-literal]
        _const_math__bits__max_u64 = ((u64)(18446744073709551615));
                                            ^
/private/var/folders/hc/7dvjblp92bg_kkpnksn4cqvr0000gn/T/v/vpm.tmp.c:13482:409: warning: integer literal is too large to be represented in a signed integer type, interpreting as unsigned [-Wimplicitly-unsigned-literal]
                ((u64)(1)), ((u64)(10)), ((u64)(100)), ((u64)(1000)), ((u64)(10000)), ((u64)(100000)), ((u64)(1000000)), ((u64)(10000000)), ((u64)(100000000)), ((u64)(1000000000)), ((u64)(10000000000)), ((u64)(100000000000)), ((u64)(1000000000000)), ((u64)(10000000000000)), ((u64)(100000000000000)), ((u64)(1000000000000000)), ((u64)(10000000000000000)), ((u64)(100000000000000000)), ((u64)(1000000000000000000)), ((u64)(10000000000000000000)), 
...
==================
(Use `v -cg` to print the entire error message)

builder error: C error.

Please make sure that:
- You have all V dependencies installed.
- You did not declare a C function that was not included. (Try commenting your code that involves C interop)
- You are running the latest version of V. (Try running `v up` and rerunning your command)

If you're confident that all of the above is true, please try running V with the `-cg` option which enables more debugging capabilities.‘```

**Comment by hungrybluedev** [comment]
Currently, there are some issues with the JSON  decoder. It is being worked upon. Once that is fixed, I assume this problem will be fixed as well.

**Comment by jagtesh** [comment]
Amazing, thank you. Something that came up in my research: https://stackoverflow.com/questions/35136202/how-do-i-mute-this-error-integer-literal-is-too-large-to-be-represented-in-a/35136303

**TL;DR** The generated C code may need additional hints for the compiler when unsigned integers are used. I'm not a C programmer so this would have been my best guess.

**Comment by nedpals** [comment]
Closed. Duplicate of #4256. Also, JSON decoding is in the works: https://github.com/vlang/v/commit/9823d80e0c9c200c9ae167cba2baac4585cd0a57

**Comment by medvednikov** [comment]
Fixed.
